packages/core/CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [3.53.0](https://github.com/mondaycom/vibe/compare/@vibe/core@3.52.2...@vibe/core@3.53.0) (2025-07-01)
```

Comment:
notice you'll probably need to rebase to get rid of this kind of unrelated changes (I **think** it would solve it)